tsk2/tsk2/Factory.swift

```diff
@@ -70,7 +124,7 @@ class Factory {
 }
 
 
-protocol DillerProtocol: AnyObject {
+protocol DelegateProtocol: AnyObject {
```

Comment:
Важно придерживаться такого нейминга чтобы всегда было понятно чьим делегатом является этот тип протокола

```suggestion
protocol FactoryDelegate: AnyObject {
```